include/deal.II/fe/fe_base.h

```diff
@@ -391,6 +400,12 @@ class FiniteElementData
   unsigned int
   n_dofs_per_cell() const;
 
+  /**
+   * @copydoc non_local_dofs_per_cell
```

Comment:
Surely you can't use `copydoc` here. The description should begin with something like `Return the...`. I would advocate referencing the `non_local_dofs_per_cell` member variable to prevent one from having to describe for a second time what a non-local variable is.